javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,16 @@
+var fullName = 'Jc Leon';
+var isCool = true;
+var totalPets = 3;
+var unknown = null;
+var never;
+console.log('value of fullName: ' + fullName);
+console.log('value of isCool: ' + isCool);
+console.log('value of totalPets: ' + totalPets);
+console.log('typeof fullName: ' + typeof fullName);
+console.log('typeof isCool: ' + typeof isCool);
+console.log('typeof totalPets: ' + typeof totalPets);
+console.log('value of unknown: ' + unknown);
+console.log('typeof unknown: ' + typeof unknown);
+console.log('value of unknown: ' + unknown);
+console.log('value of never: ' + never);
+console.log('typeof never: ' + typeof never);
```

Comment:
You should use commas instead of `+` so that you are not concatenating your label to the value.